datastore/src/main/kotlin/com/openlattice/collaborations/controllers/CollaborationController.kt

```diff
@@ -0,0 +1,189 @@
+        collaborationService.addOrganizationIdsToCollaboration(id, organizationIds)
+    }
+
+    @Timed
+    @DeleteMapping(value = [ID_PATH_PARAM + ORGANIZATIONS_PATH], consumes = [MediaType.APPLICATION_JSON_VALUE])
+    override fun removeOrganizationIdsFromCollaboration(@PathVariable(ID) id: UUID, @RequestBody organizationIds: Set<UUID>) {
+        ensureOwnerAccess(AclKey(id))
+        collaborationService.removeOrganizationIdsFromCollaboration(id, organizationIds)
+    }
+
+    @Timed
+    @GetMapping(value = [ID_PATH_PARAM + DATABASE_PATH], produces = [MediaType.APPLICATION_JSON_VALUE])
+    override fun getCollaborationDatabaseInfo(@PathVariable(ID) id: UUID): OrganizationDatabase {
+        ensureReadAccess(AclKey(id))
+        return collaborationService.getDatabaseInfo(id)
+    }
+
+    @Timed
+    @PatchMapping(value = [ID_PATH_PARAM, DATABASE_PATH], consumes = [MediaType.APPLICATION_JSON_VALUE])
+    override fun renameDatabase(@PathVariable(ID) id: UUID, @RequestBody newDatabaseName: String) {
+        ensureOwnerAccess(AclKey(id))
+        collaborationService.renameDatabase(id, newDatabaseName)
+
+    }
+
+    @Timed
+    @GetMapping(value = [ID_PATH_PARAM + PROJECT_PATH + ORGANIZATION_ID_PATH_PARAM + TABLE_ID_PATH_PARAM])
+    override fun projectTableToCollaboration(
+            @PathVariable(ID) collaborationId: UUID,
+            @PathVariable(ORGANIZATION_ID) organizationId: UUID,
+            @PathVariable(TABLE_ID) tableId: UUID
+    ) {
+        ensureReadAccess(AclKey(collaborationId))
+        ensureReadAccess(AclKey(organizationId))
+        ensureOwnerAccess(AclKey(tableId))
+
+        collaborationService.projectTableToCollaboration(collaborationId, organizationId, tableId)
+    }
+
+    @Timed
+    @DeleteMapping(value = [ID_PATH_PARAM + PROJECT_PATH + ORGANIZATION_ID_PATH_PARAM + TABLE_ID_PATH_PARAM])
+    override fun removeProjectedTableFromCollaboration(
+            @PathVariable(ID) collaborationId: UUID,
+            @PathVariable(ORGANIZATION_ID) organizationId: UUID,
+            @PathVariable(TABLE_ID) tableId: UUID
+    ) {
+        ensureReadAccess(AclKey(collaborationId))
+        ensureReadAccess(AclKey(organizationId))
+        ensureOwnerAccess(AclKey(tableId))
+
+        collaborationService.removeProjectedTableFromCollaboration(collaborationId, organizationId, tableId)
+
+    }
+
+    @Timed
+    @GetMapping(value = [ORGANIZATIONS_PATH + ORGANIZATION_ID_PATH_PARAM + TABLES_PATH], produces = [MediaType.APPLICATION_JSON_VALUE])
+    override fun getProjectedTablesInOrganization(@PathVariable(ORGANIZATION_ID) organizationId: UUID): Map<UUID, List<UUID>> {
+        // auth checks on org and collabs done inside here
+        val collaborationIds = getCollaborationsIncludingOrganization(organizationId).map { it.id }
+
+        return collaborationService.getProjectedTableIdsInCollaborationsAndOrganizations(collaborationIds, setOf(organizationId)) {
+            it.key.collaborationId
+        }
```

Comment:
This returns `Map<UUID, List<UUID>>` where the key is the table id and the value is the collaboration ids?

Reply:
organizationId -> tableIds -- also in the javadoc.